app/export.go

```diff
@@ -118,26 +173,56 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, opts
 	if err != nil {
 		return err
 	}
-
+	exportFilename := a.GetID() + "_export.log"
+	var missingAttachments []*LogFormat
 	if opts.IncludeAttachments {
-		ctx.Logger().Info("Bulk export: exporting file attachments")
+		mlog.Info("Bulk export: exporting file attachments")
 		for _, attachment := range attachments {
-			if err := a.exportFile(outPath, *attachment.Path, zipWr); err != nil {
+			if err := a.ExportAttachments(attachment, &missingAttachments, outPath, zipWr); err != nil {
 				return err
 			}
 		}
 		for _, attachment := range directAttachments {
-			if err := a.exportFile(outPath, *attachment.Path, zipWr); err != nil {
+			if err := a.ExportAttachments(attachment, &missingAttachments, outPath, zipWr); err != nil {
 				return err
 			}
 		}
-		for _, emojiPath := range emojiPaths {
-			if err := a.exportFile(outPath, emojiPath, zipWr); err != nil {
+		for _, emoji := range emojiList {
+			isExist, err := a.FileExists(*emoji.Path)
+			if err != nil {
+				return err
+			}
+			if !isExist {
+				missingAttachments = append(missingAttachments, &LogFormat{
+					Id:        emoji.Id,
+					CreatorId: emoji.CreatorId,
+					CreateAt:  time.UnixMilli(emoji.CreateAt),
+					UpdateAt:  time.UnixMilli(emoji.UpdateAt),
+					Path:      *emoji.Path,
+					Name:      *emoji.Name,
+				})
+				mlog.Warn(fmt.Sprintf("attachment %v not found and will not be exported", *emoji.Path))
+				continue
+			}
+			if err := a.exportFile(outPath, *emoji.Path, zipWr); err != nil {
 				return err
 			}
 		}
 	}
-
+	if len(missingAttachments) != 0 {
+		file, err := os.OpenFile(filepath.Join("./data/export/", exportFilename), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
```

Comment:
```suggestion
		file, err := os.OpenFile(filepath.Join(outPath, exportFilename), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
```

Reply:
It should be `./data/export/` because outPath is just `export/` and i have tested as well , it is not working